src/briefcase/commands/python_keywords_list.py

```diff
@@ -0,0 +1,35 @@
+# Create a list of reserved keywords
+pythonKeywordsList = [
```

Comment:
1. This doesn't need to be a separate file; it's a list of constants, so it could be included in the file where it's used.
2. We need to check for non-Python keywords too - Java, C, Objective C and Javascript keywords should all be on this list, because of the potential for collisions in language backends. There will be a lot of overlap with the Python keywords (as these languages all share some heritage), but there are a couple of new ones - "throw" and "do" are the two obvious ones that stood out to me, but there might be some others.
3. Since it *is* a list of constants, convention is to flag this by making the name of the variable `ALL_CAPS`.
4. Python code style also tends towards "snake_case" rather than "camelCase" when there are multiple words in a variable or function name.
5. To help work out whether words are (or aren't) in this list, it would help if they were sorted alphabetically.
6. The code you have here is syntactically correct, but stylistically, Python code style usually has the closing brace/bracket/parenthesis at the same level as the indentation of the line that started it: i.e.,
```
long_list = [
    value,
    value,
]
```